src/property-dictionary.js

```diff
@@ -50,14 +50,14 @@
     return (this._listSeparator.hasOwnProperty(property));
   };
 
-  PropertyDictionary.prototype.getValidStyleValuesArray = function(property) {
+  PropertyDictionary.prototype.supportedStyleValues = function(property) {
```

Comment:
Trying to be less verbose here. I didn't update the tests for this naming change yet.